.github/workflows/dapr.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         target_arch: ["amd64"]
     env:
       GOLANGCILINT_VER: "v1.61.0"
-      PROTOC_VERSION: "24.4"
+      PROTOC_VERSION: "25.4"
```

Comment:
Just a question: How do we decide which version we're updating to? The latest stable version seems to be 28.2, with an RC1 for 29 already out there.

Reply:
I assumed because we want to keep the same major version (`4`).